mautic/mautic
# Support EU region for sparkpost

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [N]
| New feature/enhancement? (use the a.x branch)      | [Y]
| Deprecations?                          | [N]
| BC breaks? (use the c.x branch)        | [N]
| Automated tests included?              | [Y] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This will allow user to configure SparkPost with EU region.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Config option and choose "Email Setting"
3. Select "Sparkpost in "Service to send mail through ".
4. You should able to see Dropdown to choose Sparkpost Region.
5. Send should be sent through that SparkPost region.
6. Same thing will apply while installing Mautic.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11281"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
